### PR TITLE
Show correct container runtime in init logging

### PR DESCRIPTION
**Pull request by ThorstenHans**
# Description

Init logging now logs using container runtime specified by the user, instead of always logging "Docker".

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #1209

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: pkg/standalone/standalone.go [hunk 1]
```diff
@@ -164,7 +166,8 @@ func Init(runtimeVersion, dashboardVersion string, dockerNetwork string, slimMod
 		// If --slim installation is not requested, check if docker is installed.
 		conatinerRuntimeAvailable := utils.IsDockerInstalled() || utils.IsPodmanInstalled()
 		if !conatinerRuntimeAvailable {
-			return errors.New("could not connect to Docker. Docker may not be installed or running")
+			caser := cases.Title(language.English)
+			return fmt.Errorf("could not connect to %s. %s may not be installed or running", caser.String(containerRuntime), caser.String(containerRuntime))
```
Comment (shubham1172) [inline pkg/standalone/standalone.go:170]: Do we need this? Based on the docs it seems that the two lines convert `containerRuntime` to Title-Case in English. It should be ok to display them as-is without adding two experimental libraries.

Comment (ThorstenHans) [reply pkg/standalone/standalone.go:170]: @shubham1172 I can refactor it easily to print container runtime values as they are. However `"golang.org/x/text` is already a dependency of the CLI itself

**Review by shubham1172 — APPROVED** [review]

File: pkg/standalone/standalone_test.go [hunk 2]
```diff
@@ -17,6 +17,7 @@ import (
 	"os"
 	"testing"
 
+	"github.com/dapr/cli/utils"
```
Comment (shubham1172) [inline pkg/standalone/standalone_test.go:20]: Can you please fix the lint?

Comment (ThorstenHans) [reply pkg/standalone/standalone_test.go:20]: sure

**Review by shubham1172 — APPROVED** [review]

**Review by pravinpushkar — APPROVED** [review]

**Review by mukundansundar — APPROVED** [review]
lgtm

**Pull request merged**